modules/component-store/src/component-store.ts

```diff
@@ -308,6 +331,21 @@ export class ComponentStore<T extends object> implements OnDestroy {
       });
     }) as unknown as ReturnType;
   }
+
+  callInitStateHook() {
+    this.stateSubject$
+      .pipe(
+        take(1),
+        map((val) => {
```

Comment:
Because we're not transforming the data, could we use a `tap` here and also in the other hook?

```suggestion
        tap((val) => {
```

Reply:
Yep. I only thought to inline it to avoid bringing in another operator but using tap is pretty standard